src/mame/drivers/de_3.cpp

```diff
@@ -897,16 +897,16 @@ ROM_END
 
 ROM_START(lw3_204)
 	ROM_REGION(0x10000, "maincpu", 0)
-	ROM_LOAD("lw3_cpu_c5_v2.04.bin",   0x00000, 0x10000, CRC(33cb9197) SHA1(c6b25dfd93bb5c425a606ae21f757a87a07dc320))
+	ROM_LOAD("lw3_cpu_c5_v2.04.bin", 0x00000, 0x10000, CRC(33cb9197) SHA1(c6b25dfd93bb5c425a606ae21f757a87a07dc320))
 	ROM_REGION(0x10000, "cpu3", ROMREGION_ERASEFF)
 	ROM_REGION(0x80000, "gfx3", 0)
-	ROM_LOAD("lw3_display_a2.02.rom1", 0x00000, 0x40000, CRC(4920f84f) SHA1(928e4aefdcf9462201001f4ac03d56a0cda25ec1))
-	ROM_LOAD("lw3_display_a2.02.rom0", 0x40000, 0x40000, CRC(f0ac3da3) SHA1(2e65f31e65302a5d343915488b229769f9784657))
+	ROM_LOAD("lw3_display_a2.02_rom1.u14", 0x00000, 0x40000, CRC(4920f84f) SHA1(928e4aefdcf9462201001f4ac03d56a0cda25ec1))
+	ROM_LOAD("lw3_display_a2.02_rom0.u12", 0x40000, 0x40000, CRC(f0ac3da3) SHA1(2e65f31e65302a5d343915488b229769f9784657))
 	ROM_REGION(0x010000, "soundcpu", 0)
-	ROM_LOAD("lethal_0601.u7",         0x08000, 0x08000, CRC(ba845ac3) SHA1(bb50413ace1885870cb3817edae478904b0eefb8))
+	ROM_LOAD("lethal_0601.u7", 0x08000, 0x08000, CRC(ba845ac3) SHA1(bb50413ace1885870cb3817edae478904b0eefb8))
 	ROM_REGION(0x1000000, "bsmt", 0)
-	ROM_LOAD("lethal_0601.u17",        0x00000, 0x40000, CRC(e34cf2fc) SHA1(417c83ded6637f891c8bb42b32d6898c90a0e5cf))
-	ROM_LOAD("lethal_0601.u21",        0x40000, 0x40000, CRC(82bed051) SHA1(49ddc4190762d9b473fda270e0d6d88a4422d5d7))
+	ROM_LOAD("lethal_0601.u17", 0x00000, 0x40000, CRC(e34cf2fc) SHA1(417c83ded6637f891c8bb42b32d6898c90a0e5cf))
+	ROM_LOAD("lethal_0601.u21", 0x40000, 0x40000, CRC(82bed051) SHA1(49ddc4190762d9b473fda270e0d6d88a4422d5d7))
```

Comment:
These were previously tabulated so all the offests, lengths, CRCs and SHA1s lined up from `ROM_START` to `ROM_END`.

Reply:
It was me who tabulated it on the first commit on this PR, but being the only ROM definition with the lines aligned, I thought it's better to let it like the others, and undid it, while adding the socket name on the ROM name.
You want me to re-align the lines again?